google/cloud/bigquery_v2/proto/encryption_config_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: google/cloud/bigquery_v2/proto/encryption_config.proto
-
+"""Generated protocol buffer code."""
```

Comment:
Grrrr, we've got dueling synths going here, adding and removing just this one line. @software-dov, @busunkim96, any clues why?

Reply:
Hmm, it looks like line was removed in a manual [regen](https://github.com/googleapis/python-bigquery/commit/b842fb5d1cf659a98cb317f1962804f1498a4799#diff-ae933eba62a84bf47ac5062f1684a0b7). Perhaps an older version of a dependency was used there?

Reply:
Bigtable has similar back-and-forth, with bot-generated changes only, in https://github.com/googleapis/python-bigtable/pull/111/files

Reply:
The mega-generator still uses `protoc`, right?  I'm assuming that migrating to `proto-plus` would be a V2 sort of thing.

Reply:
@tseaver Yep!  It still uses protoc. proto-plus will come along with the microgen.

Maybe protobuf itself is flip-flopping on the presence of this comment between versions? This PR was supposedly triggered by a new protobuf 
 version https://github.com/googleapis/googleapis/commit/0dc0a6c0f1a9f979bc0690f0caa5fbafa3000c2c